build-tools/scripts/BuildEverything.mk

```diff
@@ -11,7 +11,7 @@ GIT_COMMIT        = $(shell LANG=C git log --no-color --first-parent -n1 --prett
 -num-commits-since-version-change = $(shell LANG=C git log $(-commit-of-last-version-change)..HEAD --oneline 2>/dev/null | wc -l | sed 's/ //g')
 
 ZIP_OUTPUT_BASENAME   = oss-xamarin.android_v$(PRODUCT_VERSION).$(-num-commits-since-version-change)_$(OS)-$(OS_ARCH)_$(GIT_BRANCH)_$(GIT_COMMIT)
-ZIP_OUTPUT            = $(ZIP_OUTPUT_BASENAME).zip
+ZIP_OUTPUT            = $(ZIP_OUTPUT_BASENAME).tar.bz2
```

Comment:
Then perhaps we need to special-case things so macOS builds `.zip` and Linux builds `.tar.bz2`. The macOS artifacts are documented as the way for Windows users to test specific builds, and afaik Windows can't extract `.tar.bz2` artifacts. Windows *can* extract `.zip`.